HACKER/HACKER/Sources/ViewControllers/SettingViewController/Cell/SettingTableViewCell.swift

```diff
@@ -0,0 +1,52 @@
+//
+//  SettingTableViewCell.swift
+//  HACKER
+//
+//  Created by Yi Joon Choi on 2022/03/25.
+//
+
+import UIKit
+
+import SnapKit
+import Then
+
+class SettingTableViewCell: UITableViewCell {
+    
+    static let identifier = "SettingTableViewCell"
+    
+    // MARK: - UI
+    var backgroundImage = UIImageView().then {
+        $0.contentMode = .scaleAspectFill
+        $0.clipsToBounds = true
+        $0.image = UIImage(named: "blackBox")
+    }
+        
+    var titleLabel = UILabel().then {
+        $0.textColor = .hackerWhite
+        $0.font = .subtitleMedium16
+    }
+    
+    override init(style: UITableViewCell.CellStyle, reuseIdentifier: String?) {
+        super.init(style: style, reuseIdentifier: reuseIdentifier)
+        setupLayout()
+    }
+    
+    required init?(coder: NSCoder) {
+        fatalError("init(coder:) has not been implemented")
+    }
+    
+    // MARK: - Setup Method
+    private func setupLayout() {
+        addSubviews([backgroundImage,
+                     titleLabel])
+        
+        backgroundImage.snp.makeConstraints { make in
+            make.edges.equalToSuperview()
+        }
+        
+        titleLabel.snp.makeConstraints { make in
+            make.centerY.equalTo(backgroundImage.snp.centerY)
+            make.leading.equalToSuperview().inset(18)
```

Comment:
설정 같은 것들은 글자 수가 정해져있고 길이가 예상이 가능해서 괜찮은데, 
보통 라벨은 leading, trailing을 모두 지정해줘서 텍스트가 예상치 못하게 컨테이너뷰를 빠져나가는 걸 미연에 방지해줄 수도 있어!
trailing이 지정돼있어야 텍스트가 길어졌을 때 자동으로 `...` 처리가 되니까!
```suggestion
            make.leading.trailing.equalToSuperview().inset(18)
```